curriculum/challenges/spanish/08-coding-interview-prep/algorithms/implement-merge-sort.spanish.md

```diff
@@ -20,9 +20,9 @@ localeTitle: Implementar Merge Sort
 tests:
   - text: <code>mergeSort</code> es una función.
     testString: 'assert(typeof mergeSort == "function", "<code>mergeSort</code> is a function.");'
-  - text: <code>mergeSort</code> devuelve una matriz ordenada (menor a mayor).
+  - text: <code>mergeSort</code> devuelve un arreglo ordenada (menor a mayor).
```

Comment:
Maybe change it to "un arreglo ordenado" and it's good to go.

Reply:
I respectfully disagree. 
The word array  does not translates into "arreglo" in all Spanish speaking countries and it´s probably one of the worst discussions to have after the pronunciation of GIF. Depending on the books, you may find words "vector" or "matriz" used interchangeably. Scientifically speaking, a one dimension array is a multidimensional vector, or an uni-dimensional matrix. 

The closest I´ve seen as an "accepted" standard by technical books and documentation to use as a translation for array is **formación** (Vaquero, 1985). Although I consider this the correct term as well, it will probably confuse many readers as it´s rarely used outside Technical Writing.

My suggestion would be for FCC admins to check the demographics of most of the Spanish speaking users of the web, and take the most common translation for that demographic. Otherwise this pull request is doomed to be "reverted" in X months where another contributor from a different Spanish region comes and requests this to be changed to their most common use.

That, or include a glossary reference somewhere to describe this term is often translated to a different word(s)

Reply:
We focus on latin america as the demographic.

Reply:
In that case, please proceed with the change. Just leave it documented somewhere (or keep this PR id# in hand) for people like me who may come months down the line and wonder why the "weird" translation :)

Reply:
I absolutely agree and apologise for having you spend time on this. Here is a issue that I haven't yet circled around to #36829 which will cover some of this, but we may have something on the contributing guide as well to point it out.

Reply:
No worries @raisedadead I wanted to start contributing more with OSrc and this was within my expertise (before I start helping with code itself). I'm glad I could help even for a lil' amount.